.gitlab-ci.yml

```diff
@@ -32,9 +33,6 @@ test:unit:
     - git clone --depth 1 https://github.com/mendersoftware/integration
     - export INTEGRATION_DIRECTORY="$(realpath integration)"
 
-    - go list ./... | grep -v vendor | xargs -n1 -I {} go test -v -covermode=atomic -coverprofile=../../../{}/coverage.txt {} 2>&1 | tee /dev/stderr | go-junit-report > ${CI_PROJECT_DIR}/test-results.xml || exit $?
```

Comment:
This is not only update to v2 but also some `go` stuff removal:) I would propose to put it as two separated commits or add note in the PR that removed things are now just included in `v2` (my guess).

Reply:
It must be the same commit because the changes in the CI job are a consequence of the update. But you are right in that it is little justified so I am amending a bit the commit message 👍